tests/src/rhi-webgl/GLRenderTarget.test.ts

```
@@ -0,0 +1,46 @@
+      const glRenderTarget = renderTarget._platformRenderTarget as GLRenderTarget;
+
+      expect(() => {
+        glRenderTarget.activeRenderTarget(0);
+        glRenderTarget.blitRenderTarget();
+      }).not.toThrow();
+
+      renderTarget.destroy();
+    });
+  });
+
+});
```

ESLint

[error] Delete `⏎`

(prettier/prettier)